test_runner/src/main/kotlin/ftl/gc/GcAuth.kt

```diff
@@ -0,0 +1,76 @@
+package ftl.gc
+
+import com.google.api.client.auth.oauth2.Credential
+import com.google.api.client.extensions.java6.auth.oauth2.AuthorizationCodeInstalledApp
+import com.google.api.client.extensions.jetty.auth.oauth2.LocalServerReceiver
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.http.HttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.client.util.store.FileDataStoreFactory
+import ftl.config.FtlConstants
+import java.io.File
+import java.io.IOException
+
+// https://github.com/googleapis/google-oauth-java-client
+// Code from: https://developers.google.com/sheets/api/quickstart/java
```

Comment:
I'd help to add some context to the comment (like why you chose to copy from sheets)

Reply:
I updated the comment. The usage of `GoogleAuthorizationCodeFlow` is the same for any Google service. When the API changes, Google updates their official reference documentation.